error/s2n_errno.h

```diff
@@ -210,6 +210,7 @@ typedef enum {
     S2N_ERR_INVALID_SERIALIZED_SESSION_STATE,
     S2N_ERR_SERIALIZED_SESSION_STATE_TOO_LONG,
     S2N_ERR_SESSION_ID_TOO_LONG,
+    S2N_ERR_APPLICATION_BLOCKED,
```

Comment:
Is this file part of the public API? What happens if S2N_ERR_APPLICATION_BLOCKED's value changes?

Reply:
Nope. It is an internal error code used in s2n. I think we only expose s2n_error_type to public https://github.com/awslabs/s2n/blob/master/api/s2n.h#L56